samples/sensor/apds9960/sample.yaml

```diff
@@ -2,14 +2,29 @@ sample:
   name: APDS9960 sample
 tests:
   test:
-    harness: sensor
+    harness: console
     platform_whitelist: reel_board
     tags: sensors
     depends_on: i2c gpio
+    harness_config:
+        type: multi_line
+        ordered: true
+        regex:
+            - "APDS9960 sample application"
+            - "ambient light intensity (*.) proximity (*.)"
```

Comment:
You've reversed the `.*` and are missing a comma. The pattern should be: `ambient light intensity (.*), proximity (.*)`